src/Solnet.Rpc/SolanaRpcClient.cs

```diff
@@ -1056,5 +1056,17 @@ public RequestResult<ulong> GetMinimumLedgerSlot()
                 .Result;
 
         #endregion
+
+        #region Batch
+
+        public async Task<RequestResult<JsonRpcBatchResponse>> SendBatchRequestAsync(JsonRpcBatchRequest reqs)

```

Comment:
Missing comment!

Reply:
This functionality is provided via the `JsonRpcClient` base class as this was just a forwarding method and was incorrectly named. I've removed this method from `JsonRpcClient` and renamed the `SendBatchRequest` on the `JsonRpcClient` base class to `SendBatchRequestAsync`.